src/controllers/controllermanager.cpp

```diff
@@ -37,6 +37,12 @@ const int kPollIntervalMillis = 5;
 const int kPollIntervalMillis = 1;
 #endif
 
+// Note: A standard Midi device runs at 31.25 kbps,
+// a usual Midi message has 30 bits so we may not expect more than
+// 1000 messages per second
+// PortMidiController uses a buffer of 64 messages which results in
+// Maximum polling interval of 64 ms a theoritical minimum of 1 ms
+
```

Comment:
This isn't entirely true, especially in the case of DJ controllers that send tons of messages with high-res wheels. While that's the MIDI spec, many devices ignore the baud rate limits and send at wire speed (as fast as the USB bus will allow. That's a minimum of 12Mbps for USB 1.1.) We need to handle messages as fast as possible.